inkyshot/update-display.py

```diff
@@ -362,10 +362,10 @@ def temp_to_str(temp, scale):
     while message_does_not_fit == True:
         test_message = ""
         message_width = 0
-        FONT_SIZE -= 1
+        FONT = ImageFont.truetype(FONT_SELECTED, FONT_SIZE)
 
-        if FONT_SIZE <= 17:
-            FONT_SIZE = 8
+        if FONT_SIZE <= 12:
+            FONT_SIZE = 10
```

Comment:
I've been testing and I think here you might be making changes specific to the Waveshare display, the pixel font in size 10 on the InkyPHAT isn't readable at all, and the same goes for allowing the selected font to go down to 12, probably because the display is a slightly different resolution. I'd be happy to merge the optimisations but we'll need to make them specific to that display when it comes to sizes. We could also consider default values specific to the display type, but user-configurable too.